installer/x86_64/install.sh

```diff
@@ -489,11 +489,14 @@ set timeout=5
 
 EOF
 
-# Add the logic to support grub-reboot
+# Add the logic to support grub-reboot and grub-set-default
 cat <<EOF >> $grub_cfg
 if [ -s \$prefix/grubenv ]; then
   load_env
 fi
+if [ "\${saved_entry}" ] ; then
```

Comment:
>saved_entry [](start = 9, length = 11)

Where does it come from? Could you add some comment in this pull request about the problem and solution summary?